sktime/forecasting/model_selection/_split.py

```diff
@@ -673,8 +673,11 @@ class CutoffSplitter(BaseSplitter):
     which using the notation provided in :class:`BaseSplitter`,
     can be written as :math:`\{k_1,\ldots,k_n\}` for integer based indexing,
     or :math:`\{t(k_1),\ldots,t(k_n)\}` for datetime based indexing.
-    Training window's last point is equal to the cutoff,
-    while test window starts from the next observation in `y`.
+    Training window's last point is equal to the cutoff.
+    Test window is defined by forecasting horizons
+    relative to the end of the training window.
+    It will contain as many indices
+    as there are forecasting horizons provided to the `fh` argument.
```

Comment:
Hm, I would be more precise with this one:

... as there are forecasting horizon elements in the `fh` argument.
For a forecasating horizon :math:`\{h_1,\ldots, h_H\}`, the training window will
consist of the indices :math:`\{k_n+h_1,\ldots, k_n+h_H\}`.

Reply:
Thanks! Added this sentence to all splitters.

Reply:
is it a true sentence, for all the splitters?

Reply:
Yes